ipl.lisp

```diff
@@ -2,18 +2,24 @@
 ;;;; This code is licensed under the MIT license.
 
 (in-package :cl-user)
+(eval-when (:compile-toplevel :load-toplevel :execute)
+  #.(with-open-file (in "ipl-configuration.lisp")
+      `(progn ,@(loop :for form = (read in nil in)
+                      :until (eq form in)
+                      :collect form))))
+
 
 ;; Fast eval mode.
 (setf sys.int::*eval-hook* 'mezzano.fast-eval:eval-in-lexenv)
 
 ;; Host where the initial system is kept.
-;; Change the IP to the host computer's local IP.
-(mezzano.file-system.remote:add-simple-file-host :remote '(192 168 0 4))
-;; Use PATHNAME instead of #p because the cross-compiler doesn't support #p.
-;; Point *DEFAULT-PATHNAME-DEFAULTS* at the full path to the source tree.
-(setf *default-pathname-defaults* (pathname "REMOTE:/Full/path/to/Mezzano/"))
-;; Point MEZZANO.FILE-SYSTEM::*HOME-DIRECTORY* at the home directory containing the libraries.
-(setf mezzano.file-system::*home-directory* (pathname "REMOTE:/Full/path/to/Mezzano/home/"))
+(mezzano.file-system.remote:add-simple-file-host :remote *file-server-ip*)
+;; (setf *default-pathname-defaults*            #.(parse-namestring *file-server-source-directory* "REMOTE"))
+;; (setf mezzano.file-system::*home-directory*  #.(parse-namestring *file-server-home-directory*   "REMOTE"))
+(setf *default-pathname-defaults*            (make-pathname :host "REMOTE"
+                                                            :directory '(:absolute "Users" "pjb" "src" "Mezzano")))
```

Comment:
> On 25 Feb 2015, at 04:08, David T. Crosby <notifications@github.com> wrote:
> 
> In ipl.lisp <https://github.com/froggey/Mezzano/pull/5#discussion_r25315209>:
> 
> >  
> >  ;; Fast eval mode.
> >  (setf sys.int::*eval-hook* 'mezzano.fast-eval:eval-in-lexenv)
> >  
> >  ;; Host where the initial system is kept.
> > -;; Change the IP to the host computer's local IP.
> > -(mezzano.file-system.remote:add-simple-file-host :remote '(192 168 0 4))
> > -;; Use PATHNAME instead of #p because the cross-compiler doesn't support #p.
> > -;; Point *DEFAULT-PATHNAME-DEFAULTS* at the full path to the source tree.
> > -(setf *default-pathname-defaults* (pathname "REMOTE:/Full/path/to/Mezzano/"))
> > -;; Point MEZZANO.FILE-SYSTEM::*HOME-DIRECTORY* at the home directory containing the libraries.
> > -(setf mezzano.file-system::*home-directory* (pathname "REMOTE:/Full/path/to/Mezzano/home/"))
> > +(mezzano.file-system.remote:add-simple-file-host :remote *file-server-ip*)
> > +;; (setf *default-pathname-defaults*            #.(parse-namestring *file-server-source-directory* "REMOTE"))
> > +;; (setf mezzano.file-system::*home-directory*  #.(parse-namestring *file-server-home-directory*   "REMOTE"))
> > +(setf *default-pathname-defaults*            (make-pathname :host "REMOTE"
> > +                                                            :directory '(:absolute "Users" "pjb" "src" "Mezzano")))
> This would need to change back to the default /Full/path/to/Mezzano/home I think - might confuse folks otherwise
> 

In the mean time yes.  
I've not been successful in extracting those parameters from the ipl.lisp file due to the requiments of this bootstrapping process.

-- 
__Pascal J. Bourguignon__